docs/custom-commands/reponsetype.md

```diff
@@ -0,0 +1,26 @@
+---
+id: Command-Response-Type
+sidebar_position: 3
+---
+
+# Response Type
+
+Response type is how Fossabot will reply when a command is triggered.
```

Comment:
This seems like bad eng, maybe `Response type defines how Fossabot answers commands.`